src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -27,113 +203,1114 @@ const ActorInit En_Test3_InitVars = {
+    }
+}
+
+void func_80A40908(EnTest3* this, PlayState* play) {
+    if (Actor_ProcessTalkRequest(&this->player.actor, &play->state)) {
+        func_80A3E7E0(this, func_80A4084C);
+        this->player.unk_730 = &GET_PLAYER(play)->actor;
+        gSaveContext.save.weekEventReg[51] |= 8;
+        func_80151BB4(play, 0x19);
+        func_80151BB4(play, 2);
+    } else {
+        func_800B8500(&this->player.actor, play, 9999.9f, 9999.9f, PLAYER_AP_MINUS1);
+        this->unk_D78 = &D_80A41854[6];
+        this->player.actor.textId = this->unk_D78->textId;
+        this->player.actor.flags |= (ACTOR_FLAG_8 | ACTOR_FLAG_1);
+    }
+}
+
+void func_80A409D4(EnTest3* this, PlayState* play) {
+    if ((play->actorCtx.unk5 & 0x20) || (play->actorCtx.unk5 & 0x10)) {
+        play->actorCtx.unk5 &= ~0x10;
+        func_80A3F0B0(this, play);
+        ActorCutscene_SetReturnCamera(CAM_ID_MAIN);
+    } else {
+        sInput = *CONTROLLER1(&play->state);
+    }
+}
+
+void func_80A40A6C(EnTest3* this, PlayState* play) {
+    gSaveContext.save.weekEventReg[64] |= 0x20;
+}
+
+void EnTest3_Update(Actor* thisx, PlayState* play2) {
+    PlayState* play = play2;
+    EnTest3* this = THIS;
+
+    sInput.rel.button = sInput.cur.button;
+    sInput.cur.button = 0;
+    sInput.rel.stick_x = 0;
+    sInput.rel.stick_y = 0;
+
+    play->actorCtx.unk5 &= ~0x80;
+    this->player.actor.draw = EnTest3_Draw;
+    D_80A41D48 = false;
+    this->player.actor.flags &= ~(ACTOR_FLAG_8 | ACTOR_FLAG_1);
+    if (Cutscene_CheckActorAction(play, 0x1FA) && !((this->player.actor.category == ACTORCAT_PLAYER) &&
+                                                    ((play->actorCtx.unk5 & 0x20) || (play->actorCtx.unk5 & 0x10)))) {
+        if (this->player.csMode != 5) {
+            play->startPlayerCutscene(play, &this->player, 5);
+        }
+        play->actorCtx.unk5 &= ~0x10;
+    } else if (this->player.actor.category == ACTORCAT_PLAYER) {
+        func_80A409D4(this, play);
+    } else if (play->startPlayerCutscene(play, &this->player, 0)) {
+        if (this->unk_D88 >= 7) {
+            Vec3f worldPos;
+
+            Math_Vec3f_Copy(&worldPos, &this->player.actor.world.pos);
+            this->unk_D80 = 4;
+            func_80A40230(this, play);
+            Math_Vec3f_Copy(&this->player.actor.world.pos, &worldPos);
+            D_80A41D48 = false;
+            this->unk_D84 = 0.0f;
+        }
+    } else {
+        D_80A41D40 = 0.0f;
+        D_80A41D44 = this->player.actor.shape.rot.y;
+        this->unk_D94(this, play);
+        sInput.press.button = (sInput.rel.button ^ sInput.cur.button) & sInput.cur.button;
+        func_800B6F20(play, &sInput, D_80A41D40, D_80A41D44);
+    }
+    play->playerUpdate(&this->player, play, &sInput);
+    if (D_80A41D48) {
+        this->player.actor.world.pos.x = D_80A41D50.x;
+        this->player.actor.world.pos.z = D_80A41D50.z;
+        this->player.linearVelocity = 0.0f;
+    }
+}
+
+static s32 D_80A418C8 = false;
```

Comment:
```suggestion
s32 D_80A418C8 = false;
```
Don't need the static if it's not named yet